parse_result.py

```diff
@@ -103,6 +103,37 @@ def has_passed(job):
         return False
     return job["result"]["status"] in { 0, "0", "pass" }
 
+
+def create_badge(filename, status="failed"):
```

Comment:
The badge should be able to handle states other than `failed` and `passed`, but ok ;-).